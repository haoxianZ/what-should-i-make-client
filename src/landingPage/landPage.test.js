import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import LandingPage from './landingPage';

describe(`landing page component`, () => {
  it('renders without crash', () => {
    const wrapper = shallow(<LandingPage />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
}) 