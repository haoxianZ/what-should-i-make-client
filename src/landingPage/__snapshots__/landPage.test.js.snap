// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`landing page component renders without crash 1`] = `
<div
  className="landingPage"
>
  <Link
    style={
      Object {
        "textDecoration": "none",
      }
    }
    to="/home-page"
  >
    <header>
      What Should I Make?
    </header>
  </Link>
  <nav>
    <Link
      style={
        Object {
          "textDecoration": "none",
        }
      }
      to="/about"
    >
      About
    </Link>
    <Link
      style={
        Object {
          "textDecoration": "none",
        }
      }
      to="/contact"
    >
      Contact
    </Link>
  </nav>
  <main
    className="intro"
  >
    <h1>
      Welcome!
    </h1>
    <h3>
      What is this?
    </h3>
    <h4>
      This is an app to help you to decide what to cook base on what is in your fridge.
    </h4>
    <h3>
      How do I use this App?
    </h3>
    <h4>
      Create an account and log in what you have in the Fridge. Then explore what you can cook with them!
    </h4>
    <Link
      style={
        Object {
          "textDecoration": "none",
        }
      }
      to="/home-page"
    >
      <button
        className="start"
      >
        Let's get started
      </button>
    </Link>
  </main>
</div>
`;
